core/src/lib.rs

```diff
@@ -291,12 +339,29 @@ impl Schema {
     pub fn identifies_attribute(&self, x: &NamespacedKeyword) -> bool {
         self.get_entid(x).map(|e| self.is_attribute(e)).unwrap_or(false)
     }
+
+    /// Returns an symbolic representation of the schema suitable for applying across Mentat stores.
+    pub fn to_edn_value(&self) -> edn::Value {
+        edn::Value::Vector((&self.schema_map).into_iter()
```

Comment:
so, without the `&` in front of the `self.schema_map` call, I get the following error:
```
file: 'file:///Users/emilytoop/Development/mentat/core/src/lib.rs'
severity: 'Error'
message: 'E0507: cannot move out of borrowed content
  label: cannot move out of borrowed content'
at: '345,29'
source: '
```

Turning the `schema_map` into a reference (like I did when I used it in the loop previously) was the only way I could then call `into_iter` without that error. I am not sure how else I should have done it. Looking at all other references to `into_iter` elsewhere in the code, it is always either called on a reference to an object, or directly on a locally created object. If I have misunderstood something fundamental around ownership and references here, please let me know what it is!